tailwind.config.ts

```diff
@@ -55,25 +194,71 @@ const config: Config = {
     },
     extend: {
       colors: {
+        ...colorPalette,
         ...newColors,
-        primary: colors.blue,
-        secondary: colors.gray,
-        success: colors.green,
-        error: colors.red,
-        warning: colors.orange,
-        info: colors.sky,
+        primary: {
+          DEFAULT: colorPalette.blue['05'],
+          active: colorPalette.blue['05'],
+          hover: colorPalette.blue['06'],
+          inactive: colorPalette.blue['02']
+        },
+        secondary: {
+          DEFAULT: colorPalette.gray['07'],
+          active: colorPalette.gray['07'],
+          hover: colorPalette.gray['08'],
+          inactive: colorPalette.gray['04']
+        },
+        success: {
+          DEFAULT: colorPalette.green['05'],
+          active: colorPalette.green['05'],
+          hover: colorPalette.green['06'],
+          fill: colorPalette.green['01']
+        },
+        error: {
+          DEFAULT: colorPalette.red['05'],
+          active: colorPalette.red['05'],
+          hover: colorPalette.red['06'],
+          fill: colorPalette.red['01']
+        },
+        warning: {
+          DEFAULT: colorPalette.orange['05'],
+          active: colorPalette.orange['05'],
+          hover: colorPalette.orange['06'],
+          fill: colorPalette.orange['01']
+        },
+        info: {
+          DEFAULT: colorPalette.blue['05'],
+          active: colorPalette.blue['05'],
+          hover: colorPalette.blue['06'],
+          fill: colorPalette.blue['01']
+        },
+        background: {
+          level1: colorPalette.white,
+          level2: colorPalette.gray['01'],
+          level3: colorPalette.gray['02'],
+          level4: colorPalette.gray['03'],
+        },
+        panel: newColors['black-pearl'],
+        surface: newColors['charade'],
+        typography: {
```

Comment:
Is this a standard field?

Reply:
It's my vision to name colors for text.
Figma has `text-01, text-02, text-03` currently
I've shared my vision with Lisa to rethink how to mane it and make it like
```
typography: {
  title: 'color',
  label: 'color'
  body: 'color'
  button: 'color'
  etc...
}
```